contrib/test/integration/build/cri-tools.yml

```diff
@@ -4,7 +4,7 @@
   git:
     repo: "https://github.com/kubernetes-sigs/cri-tools.git"
     dest: "{{ ansible_env.GOPATH }}/src/github.com/kubernetes-sigs/cri-tools"
-    version: v1.20.0
+    version: v1.21.0
```

Comment:
Why bump to 1.21 for release-1.20?

Reply:
to get https://github.com/kubernetes-sigs/cri-tools/pull/733 easier :)